gulpfile.js

```diff
@@ -335,12 +335,23 @@ gulp.task('deploy-s3', function(done) {
 
 });
 
+function getCredentials() {
```

Comment:
This function can be replaced with 
```javascript
var getCredentials = Promise.promisify(aws.config.getCredentials, {context: aws.config})
```

You might not need the context options, it depends on how config is implemented.